api/transaction/routes.go

```diff
@@ -68,7 +67,6 @@ func Routes(router *wrapper.RouterWrapper) {
 	router.RegisterHandler(http.MethodPost, "/cost", ComputeTransactionGasLimit)
 	router.RegisterHandler(http.MethodPost, "/send-multiple", SendMultipleTransactions)
 	router.RegisterHandler(http.MethodGet, "/:txhash", GetTransaction)
-	router.RegisterHandler(http.MethodGet, "/:txhash/status", GetTransactionStatus)
```

Comment:
Maybe remove the entry from api.toml and associated config struct?

Reply:
done.